database/rrd.h

```diff
@@ -974,10 +975,8 @@ struct rrdhost {
     uuid_t  host_uuid;                              // Global GUID for this host
     uuid_t  *node_id;                               // Cloud node_id
 
-#ifdef ENABLE_HTTPS
-    struct netdata_ssl ssl;                         //Structure used to encrypt the connection
-    struct netdata_ssl stream_ssl;                         //Structure used to encrypt the stream
-#endif
+    struct netdata_ssl ssl;                         // Structure used to encrypt the connection
+    struct netdata_ssl stream_ssl;                  // Structure used to encrypt the stream
```

Comment:
These variables were inside `ENABLE_HTTPS` for users have conditions to compiled netdata with next two options:

```sh
--disable-https            Explicitly disable TLS support.
--disable-dbengine         Explicitly disable DB engine support.
```
Now we will force them to have `openssl` or analogous `TLS` library. If this is true, probably we need to remove at least `--disable-https` from options. (ping @Ferroin ).